### PR TITLE
telemetry: fix the bug of annoying log caused by telemetry

### LINKED ISSUE #25760: Annoying telemetry log
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

```
genius@genius-System-Product-Name:~/project/src/github.com/pingcap/tidb$ ./bin/tidb-server -V
[2021/06/25 15:03:04.812 +08:00] [INFO] [data_slow_query.go:146] ["Telemetry slow query stats initialized"] [currentSQBInfo={65.536:0,32.768:0,262.144:0,4194.304:0,8388.608:0,16777.217:0,0.008:0,0.016:0,0.064:0,0.002:0,1.024:0,67108.867:0,2097.152:0,33554.434:0,+Inf:0,0.032:0,0.128:0,0.256:0,0.512:0,4.096:0,0.004:0,16.384:0,1048.576:0,8.192:0,131.072:0,524.288:0,0.001:0,2.048:0,134217.734:0] [lastSQBInfo={0.032:0,1.024:0,262.144:0,16777.217:0,0.004:0,8.192:0,524.288:0,1048.576:0,8388.608:0,2.048:0,67108.867:0,134217.734:0,4194.304:0,16.384:0,32.768:0,33554.434:0,0.008:0,0.512:0,4.096:0,65.536:0,0.001:0,0.064:0,0.128:0,131.072:0,2097.152:0,0.016:0,0.256:0,+Inf:0,0.002:0]
Release Version: v5.1.0
Edition: Community
Git Commit Hash: 8acd5c88471cb7b4d4c4a8ed73b4d53d6833f13e
Git Branch: HEAD
UTC Build Time: 2021-06-25 07:02:42
GoVersion: go1.16.5
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false
```

What the hell is this `[2021/06/25 14:59:09.994 +08:00] [INFO] [data_slow_query.go:146]`, and also, when I running a tidb server locally with unistore, there are some annoying telemetry log, too.

### 2. What did you expect to see? (Required)

No annoying log

### 3. What did you see instead (Required)

The annoying log

### 4. What is your TiDB version? (Required)

origin/release-5.1 latest

Release Version: v5.1.0
Edition: Community
Git Commit Hash: 8acd5c88471cb7b4d4c4a8ed73b4d53d6833f13e
Git Branch: HEAD
UTC Build Time: 2021-06-25 07:02:42
GoVersion: go1.16.5
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false

### LINKED ISSUE #25785: TiDB reports 'querySQLMetric got error' every minute
## Development Task

Start TiDB with unistore:

```
[2021/06/28 09:30:22.187 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:31:22.693 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:32:23.198 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:33:23.702 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:34:24.207 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:35:24.712 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:36:25.216 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
[2021/06/28 09:37:25.721 +08:00] [WARN] [data_window.go:112] ["querySQLMetric got error"]
```

### PR DESCRIPTION
**Pull request by YinWeiling**
### What problem does this PR solve?

Issue Number: close #25760  close #25785 
fix the bug of sql staticis & slow query staticis

Problem Summary:
annoying log caused by telemetry, TiDB reports 'querySQLMetric got error' every minute.

### What is changed and how it works?

What's Changed:

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
use `./bin/tidb-server` to set up TiDB and see the log.

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

- fix the bug of annoying logs caused by telemetry

### CODE REVIEW

**Comment by lilyjazz** [comment]
reviewer @lucklove

**Review by wjhuang2016 — APPROVED** [review]

File: telemetry/data.go [hunk 1]
```diff
@@ -49,6 +51,10 @@ func generateTelemetryData(ctx sessionctx.Context, trackingID string) telemetryD
 	}
 
 	r.WindowedStats = getWindowData()
+	promErrCnt := promAddressErr.Load()
```
[the remark this answers is not in the document]
Comment (YinWeiling) [reply telemetry/data.go]: thx for your comment~ I will remove it~

**Review by lucklove — APPROVED** [review]
LGTM

**Pull request merged**